### DISCUSSION TITLE
My "Stars" feeds are empty

### DISCUSSION THREAD

**Discussion by chazeon** [community/community#66210] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

In the old feed, I can see other people starting projects this week, but now after GitHub "integrate" the "for you" feed with the "following" feed, there are no recent stars. Turning on only the "Star" gives me an empty feed. Is anyone else having this issue?

**Comment by dmarx** [comment]
yup, same. i tried to recreate the old "following" feed behavior by filtering down to only stars and repos, and now my feed is empty despite it showing months old activity for stuff like new follows when i had that active. 
<img width="730" alt="Screen Shot 2023-09-06 at 2 29 56 PM" src="https://github.com/community/community/assets/1466881/247e410e-1653-464c-92ee-6286d6c4b826">

**Comment by transitive-bullshit** [comment]
Agreed. This has to be a glaring and obvious bug that's existed ever since the introduction of the "For You" tab. My "Stars"-only feed is also empty.

As a temporary workaround, you can view the original dashboard feed here: https://github.com/dashboard-feed

**Reply by iainelder** [reply]
I've bookmarked the good old "dashboard feed" while GitHub fixes this. Thanks for sharing!

**Reply by iainelder** [reply]
If you're comfortable with a user script, you can use this to replace the new feed with the original dashboard.

https://github.com/wangrongding/github-old-feed

**Reply by loneicewolf** [reply]
I know this is about stars but the following thing is also the same, and not even there it updates. I have tried on another account (an account I just created to see this) I followed a person, but the 'following' isn't visible. I.e I am following that person but it's not visible. My face when I saw this:  `:huh:?`

**Reply by loneicewolf** [reply]
# basically
 what I mean is the amount of following  and followers seems to have grind to a halt. If you un follow or en follow someone, it still shows the same old amount.. 🤔

**Reply by ddelange** [reply]
how was this missed in all testing rounds?
[![Agile Development](https://user-images.githubusercontent.com/14880945/246339340-458608a0-3272-4cff-9d5b-93e108ccb8cc.gif)](https://gist.github.com/ddelange/aa4a97f5d8a9d9e3625d405161aac9b2#agile-developmenthttpsuser-imagesgithubusercontentcom14880945246339340-458608a0-3272-4cff-9d5b-93e108ccb8ccgif)

**Comment by gitbls** [comment]
yuck. This is a horrible step backwards.

**Comment by 0x676e67** [comment]
The updates suck

**Comment by nwg-piotr** [comment]
Same here. I miss my stars very much.

**Comment by sooftware** [comment]
Me too.

**Comment by iainelder** [comment]
Same here. The stars feed is one of my favorite parts of GitHub. Please fix this!

![image](https://github.com/community/community/assets/5024311/e7d2002d-3766-4eb0-90eb-fe86c8cd2f65)

**Reply by iainelder** [reply]
Using this script you can restore the old version of the feed. Thanks @wangrongding!
https://github.com/wangrongding/github-old-feed

![image](https://github.com/community/community/assets/5024311/ad59d718-6b7d-426d-bb81-a9bcbfc2f612)

**Comment by crstian19** [comment]
Same here pls fix it! Fun fact, Gitlab isn't broken

![image](https://github.com/community/community/assets/21632768/3b9b41d3-b272-44b0-b2be-efe775b22809)

**Comment by izniburak** [comment]
I have same issue. :( I hope it fix quickly.

**Comment by darky** [comment]
Now I can see starred of my repositories. But not of my followings.

**Comment by woheller69** [comment]
I also see stars now, but the stars from the last 2 days are missing.
Forks aren't shown at all

**Comment by darky** [comment]
So, following no more matters. Why follow someone if you don't see his/her activity?

**Comment by agn-7** [comment]
Apparently, In the new version of the GitHub feed, you can only see your given stars by your followers, not the repositories that get stars by your followers!

**Comment by Shpota** [comment]
I would appreciate if I could see:
- people star repositories I own
- people star repositories of organizations that I am a part of or I own

The second part didn't work in the old feed, but it would be nice to add it and maybe make it configurable.

**Comment by cambragol** [comment]
Mine too.

Just thought everything was real quiet for the past few months.

Also I don't see any activity from any of my 'followed' repositories. Not in the feed or elsewhere. Not sure if this is a related or separate issue though.

In fact, the only thing that shows up in my feed, is releases, even though every category is selected.

**Comment by GoktugYalcin** [comment]
Same issue. Are there any guy has update for this issue?
